src/openrct2/world/MoneyEffect.cpp

```diff
@@ -79,7 +79,7 @@ void rct_money_effect::Create(money32 value)
 {
     CoordsXYZ mapPosition = { gCommandPosition.x, gCommandPosition.y, gCommandPosition.z };
 
-    if (mapPosition.x == LOCATION_NULL)
+    if (mapPosition.x == COORDS_NULL)
```

Comment:
Hadn't even realised that we had `COORDS_NULL`. Some of the Coords object implement `isNull()`, should we extend to the others so that the code doesn't have to reference `COORDS_NULL`?

Reply:
Yeah i think i might need to revert this change anyway as need to check how the value is set.